text/0419-cdk-launchpads.md

```diff
@@ -0,0 +1,632 @@
+
+```shell
+$ cdk-lpads status —profile acc-456 aws://456/eu-west-2
+
+Environment - account: 456, region: eu-west-2
+1 Launchpad found:
+1 | legacy (V1) CDK bootstrap Stack
+    Run `cdk-lpads update` to upgrade it to the newest version.
+```
+
+#### 2. `new`
+
+Creates a new Launchpad in the given account and region.
+By default, the command will include a wizard that will guide the user through the options:
+
+```shell
+$ cdk-lpads new
+
+Welcome! This wizard will guide you through creating a new Launchpad in the environment.
+To start, let's make sure we're operating in the correct AWS account and region:
+
+Environment - account: 123, region: eu-west-2
+
+Does this look correct? [Y/N] Y
+
+Looks like the environment aws://123/eu-west-2 doesn't have any existing Launchpads.
+
+Do you want to customize the qualifier used? [default: hnb659fds]:
+...
+```
+
+Alternatively, the `new` command can be customized directly, instead of going through the wizard;
+see "Customizing the Launchpad" below.
+
+#### 3. `update`
+
+This command attempts to update an existing Launchpad.
+In the case it's the default CDK bootstrap Launchpad, it will simply update it to the latest version.
+The `update` command can be customized directly, same as the `new` command;
+see "Customizing the Launchpad" below.
+
+#### 4. `diff`
+
+Allows you to see what differences would be applied if you updated your Launchpad.
+Useful to run before `update` to confirm the changes are what you expect.
+
+#### 5. `init json/yaml/cdk [ts/js/java/python/csharp/go]`
+
+Similar to the `cdk init` command,
+this command creates a skeleton that allows you get started with customizing your Launchpad.
+It supports both the JSON/YAML and CDK ways of customizing Launchpads (see below).
+
+### Ways of customizing the Launchpad
+
+The default Launchpad contains the same resources as the `cdk bootstrap` command (in version 2) creates.
+A CDK Launchpad can be customized with the following additional options:
+
+1. Compliance rules.
+2. Additional resources (deployed using CloudFormation, CDK, or in any other way, and simply referenced by the Launchpad).
+3. Modifications to the CDK bootstrap resources (using either CloudFormation templates or CDK).
+4. Setting IAM Permission Boundaries used by all applications deployed into a given environment.
+
+There are two ways of customizing a Launchpad: using JSON/YAML files, or using CDK code.
+
+#### Customizing the Launchpad with JSON/YAML files
+
+This option of customizing the Launchpad is geared primarily towards platform administrators
+who don't want to use a programming language (and thus, CDK code) to customize their Launchpads.
+Using a JSON or YAML file in a specific format allows customizing all aspects of the Launchpad,
+without the need to write a single line of code.
+
+The JSON file has the basic structure (the YAML file is very similar):
+
+```json
```

Comment:
Is there a spec somewhere for this file?

Reply:
There is not. The spec is kind of gradually introduced throughout the RFC. I don't want to focus too much on the exact details of this spec, because I don't think it's that important - we can iron out those details when doing the implementation (kind of what I was alluding to during the discussion with @alexpulver in https://github.com/aws/aws-cdk-rfcs/pull/421#discussion_r862308094).

What I'm more interested in is an opinion on whether we should create a new format at all, or rather have separate files, and piggy back 100% on existing formats (what @RichiCoder1 is saying here: https://github.com/aws/aws-cdk-rfcs/pull/421#discussion_r861233400).